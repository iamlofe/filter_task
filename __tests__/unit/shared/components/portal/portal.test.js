import React from 'react';
import { shallow } from 'enzyme';

import Portal from 'shared/components/portal/portal';

describe('button-filter component', () => {
    it('should shalow rendered without crashing', () => {
        document.body.innerHTML = '';
        const element = document.createElement('div');
        element.id = 'portal-root';
        document.body.appendChild(element);
        const portalComponentP = (
            <Portal>
                <div> sss</div>
            </Portal>
        );
        const portalComponent = shallow(portalComponentP);

        expect(portalComponent).toMatchSnapshot();
    });
});
