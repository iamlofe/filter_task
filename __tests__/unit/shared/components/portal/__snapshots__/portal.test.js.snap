// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button-filter component should shalow rendered without crashing 1`] = `
<div>
  <Portal
    containerInfo={
      <div
        style="position: absolute;"
      />
    }
  >
    <div>
       sss
    </div>
  </Portal>
</div>
`;
